api.test.js

```diff
@@ -0,0 +1,13 @@
+import { countMovies } from './src/script/api.js';
+
+test('displays number of items on home-page', () => {
+  document.body.innerHTML = '<h1 class="count-movies"></h1>';
+  countMovies([1, 2, 3, 4, 5, 6, 7, 59, 89, 20]);
+  expect(document.querySelector('.count-movies').innerHTML).toBe('10');
+});
+
+test('displays nothing if there are no items on home-page', () => {
+  document.body.innerText = '<h1 class="count-movies"></h1>';
+  countMovies([]);
+  expect(document.querySelector('.count-movies').innerHTML).toBe('');
+});
```

Comment:
- [ ] These tests failed. I attached an image of the failing tests. Kindly fix it.
<img width="773" alt="chrome_gRAyyGhLH5" src="https://user-images.githubusercontent.com/81468675/180494889-0d2dfda2-f121-4bca-abef-4ad765aefb24.png">